airbyte-integrations/connectors/source-posthog/source_posthog/manifest.yaml

```diff
@@ -49,8 +46,8 @@ definitions:
   projects_stream:
     $ref: "#/definitions/base_stream"
     $parameters:
-      name: "projects"
       path: "projects"
+    name: "projects"
```

Comment:
Wanted to point out an example of where extracting `name` or (`primary_key`) from the `$parameters` block might have some unintended side effects that may break a connector. This would work for the stream, retriever, and requester components. But this is probably going to break the `schema_loader` component too because it relies on the `name` in the parameters block to build the `file_path`. 

I'm actually of the mindset that we can leave many of the connectors with the `name` and `primary_key` under the declarative stream's `$parameters` because they are technically still correct. Albeit they're not the most idiomatic, but I can see a lot of little edge cases where removing them could break something not immediately obvious.

We should still remove the fields from the requester and retriever like you are already doing because the fields no longer exist there, but I was just talking about the `$parameters` block.

Reply:
Thanks for the example @brianjlai. I've reverted the change that had removed `name` and `primary_key` from `$parameters`. I've also removed `name` and `primary_key` as top-level field for these manifests, since it's redundant, but let me know if you'd like to keep it in and I'll revert the last commit, e.g. to encourage people to put `name` and `primary_key` directly on the stream.

Reply:
sorry missed responding to this in the review. How you have it is good! Moving forward with the updated template, it should guide people to having `name` and `primary_key` at the stream level and not in `$parameters`.